modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -93,18 +102,28 @@ const convertFieldsToString = (requestedFields: RequestedFieldsMap) => {
 	return gqlFieldsString;
 };
 
+/**
+ * Query string creation
```

Comment:
This is not a useful description. Consider expanding to describe what the content of the query string will be and how that string can be used. Also useful, consider: what are the arguments and how are they used?